docs/uefi-db.md

```diff
@@ -0,0 +1,203 @@
+---
+title: UEFI Secure Boot Certificates
+---
+
+## Executive Summary
+
+On the 11th September 2025 a certificate used for signing boot media will expire. Microsoft will not sign updated boot media with the old key, and that **at least one major OEM** is not going to be shipping the expired key on new hardware. This means that existing install media may not boot on some new laptop, desktop and server devices, and that future updates to boot packages may not boot on old hardware.
+
+Microsoft is shipping fixes for select OEMs using Windows Updates automatically. The workaround for Linux is to manually disable secure boot which would be unpopular with anyone that cares about security. Using fwupd is a way that can distribute the updated certificates in Linux.
+
+## Important Terms
+
+* `PK`: “*Platform Key*” – one X509 certificate created by the OEM, e.g. Lenovo
+* `KEK`: “*Key Exchange Key*” – multiple certificates (created by Microsoft and the OEM), used to update db, signed by the vendor PK
+* `db`: “*Signature Database*” – multiple certificates (created by Microsoft and the OEM), hashes, and signatures, used to allow binaries, signed by KEK
+* `dbx`: “*Forbidden Signatures Database*” – multiple certificates, hashes, and signatures, used to block binaries, signed by KEK
+* 3rd party certificate – the certificate Microsoft uses to sign non-MS bootloaders that Linux uses, e.g. Shim
+  * Windows Production CA 2011 – the old “PCA” certificate
+  * Microsoft UEFI CA 2023 (third-party) – the new certificate
+
+## Introduction
+
+The Microsoft certificate which is used for signing “3rd party” boot media (e.g. `shimx64.efi`) will expire on the 11th September 2025. Microsoft has created a new certificate which can be used for signing now. Whilst Microsoft **may** allow us to “dual sign” the shim binary with the old and new certificate until the cut-over date, when the certificate has expired they will only sign our shim with the new 3rd party certificate. New laptop, desktop and server products from some OEMs will ship from the factory with **only** the new Microsoft Windows UEFI CA 2023 certificate. This means any shim binaries signed with the old PCA 2011 certificate will not be allowed to run.
```

Comment:
fwupd doesn't sign a shim binary, so this shouldn't be "our shim".

Reply:
Ack, fixed.